timeboost-contract/src/binaries/deploy.rs

```diff
@@ -0,0 +1,112 @@
+//! CLI for contract deployment
+//!
+//! # Usage
+//!
+//! ```
+//! # Write config to stdout
+//! cargo run --bin deploy -- -m "your mnemonic here" -i 0 -u http://localhost:8545
+//!
+//! # Write config to a file
+//! cargo run --bin deploy -- -m "your mnemonic here" -i 0 -u http://localhost:8545 -o output.toml
+//! ```
+//!
+//! # Local test
+//! Run `just test-contract-deploy`
+use alloy::{primitives::Address, providers::WalletProvider};
+use anyhow::{Context, Result};
+use clap::Parser;
+use serde::{Deserialize, Serialize};
+use std::{fs, path::PathBuf};
+use timeboost_contract::provider::build_provider;
+use timeboost_utils::types::logging;
+use tracing::info;
+use url::Url;
+
+#[derive(Clone, Debug, Parser)]
+struct Args {
+    #[clap(short, long)]
+    mnemonic: String,
+
+    #[clap(short, long)]
+    index: u32,
+
+    #[clap(short, long)]
+    url: Url,
+
+    #[clap(short, long)]
+    output: Option<PathBuf>,
+}
+
+/// Config type for the key manager who has the permission to update the KeyManager contract
+/// See `test-configs/keymanager.toml` for an example
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
```suggestion
#[derive(Debug, Serialize)]
```

These structs are no longer parsed AFAICT.